app/views/posts/index.html.erb

```diff
@@ -1,6 +1 @@
-<h1>Welcome to post index page</h1>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= post.title %></li>
-  <% end %>
-</ul>
```

Comment:
### Although I am highlighting codes from the `Posts#Index` page, the below recommendation is for the `Posts#Show` page 🔽. You didn't make any changes to the Post#Show page, so I can't highlight any code from it 🧭:

- [ ] Kindly design the `Post#Show` page to look identical to that of the wireframe provided for this project. When I manually visit the `Post#Show` page, this is what is rendered 🖱️:

![image](https://github.com/ridabensalem/blog-app/assets/77106962/16b666b9-ff37-4174-bbab-6c5116dff073)

#### Please design the `Post#Show` Page to look identical to this wireframe below 💻 :

![image](https://github.com/ridabensalem/blog-app/assets/77106962/18457b77-fb61-4886-bea3-6305b89973f5)